packages/wear-os/README.md

```diff
@@ -1,12 +1,428 @@
+> on('watchHeartRateSamplesAcquired', run('writeRecords'));
+> on('watchGeolocationSamplesAcquired', run('writeRecords'));
+>```
+> **Note**: To use the `writeRecords` task, the persistence package must be installed and configured. See [persistence package docs](../persistence/README.md).
+
+#### Stop data collection for a sensor
+To register these tasks for their use, you just need to import them and call their generator functions inside your application's task list:
+
+```typescript
+import { Task } from '@awarns/core/tasks';
+import {
+  stopDetectingWatchSensorChangesTask,
+  WatchSensor,
+} from '@awarns/wear-os';
+
+export const demoTasks: Array<Task> = [
+  stopDetectingWatchSensorChangesTask(WatchSensor.ACCELEROMETER), // stopDetectingWatchAccelerometerChanges
+  stopDetectingWatchSensorChangesTask(WatchSensor.GYROSCOPE),     // stopDetectingWatchGyroscopeChanges
+  stopDetectingWatchSensorChangesTask(WatchSensor.MAGNETOMETER),  // stopDetectingWatchMagnetometerChanges
+  stopDetectingWatchSensorChangesTask(WatchSensor.HEART_RATE),    // stopDetectingWatchHeartRateChanges
+  stopDetectingWatchSensorChangesTask(WatchSensor.GEOLOCATION),   // stopDetectingWatchGeolocationChanges
+];
+```
+
+> **Note**: a stop task of a specific `WatchSensor` can be used to stop the collection started by any start task
+> for that `WatchSensor` no matter the specific configuration.
+
+Task generator parameters:
+
+| Parameter name          | Type                    | Description                              |
+|-------------------------|-------------------------|------------------------------------------|
+| `sensor`                | `WatchSensor`           | Sensor to stop the data collection from. |
+
+Task output events:
+
+> These tasks don't produce significant events after they complete their execution aside from the regular `{task-name}Finished` events.
+>
+> Example usage in the application task graph:
+> ```ts
+> on('startEvent', run('startDetectingWatchAccelerometerChanges').every(1, 'minute'));
+> on('startEvent', run('startDetectingWatchGyroscopeChanges').every(1, 'minute'));
+> on('startEvent', run('startDetectingWatchMagnetometerChanges').every(1, 'minute'));
+> on('startEvent', run('startDetectingWatchHeartRateChanges').every(1, 'minute'));
+> on('startEvent', run('startDetectingWatchGeolocationChanges').every(1, 'minute'));
+> 
+> on('watchAccelerometerSamplesAcquired', run('stopDetectingWatchAccelerometerChanges'));
+> on('watchGyroscopeSamplesAcquired', run('stopDetectingWatchGyroscopeChanges'));
+> on('watchMagnetometerSamplesAcquired', run('stopDetectingWatchMagnetometerChanges'));
+> on('watchHeartRateSamplesAcquired', run('stopDetectingWatchMagnetometerChanges'));
+> on('watchGeolocationSamplesAcquired', run('stopDetectingWatchMagnetometerChanges'));
+>```
+> **Note**: it makes no sense to use these tasks without using before their complementary tasks to start the data collection.
+
+#### Send a message to the paired watch
```

Comment:
I think you should indicate somewhere that the messaging feature must be enabled for this to work. It's at the beginning of the docs, but probably the dev won't remember